### ISSUE TITLE
Adding "--help" and "-h" to the plugins

### ISSUE THREAD

**Issue by ghost** [rgcr/m-cli#124]
Hi there,

what about adding "--help" and "-h" to the plugins?
I'm coming from the Linux side and i recently ran "m wallpaper --help" by mistake and it changed my wallpaper to the default Sierra one.

Not so great.

What do you think about that?

**Comment by nanisean** [comment]
I just ran this expecting a help description for wallpaper and now my wallpaper is black and Dock.app won't open, ever after reboot. Now I have no dock and can't use app switcher or spaces.
No idea how to fix this on Catalina...

Not good at all.

**Comment by bensleveritt** [comment]
You're right, that's a terrible experience @nanisean. I *think* the latest release isn't nearly as destructive for the wallpaper command, and `m wallpaper -h` fails safe.

I'll have a think about how to alias `-h` and `--help` as they're common
